web-client/src/app/views/delete-user/delete-user.page.ts

```diff
@@ -40,18 +44,47 @@ export class DeleteUserPage implements OnInit {
   /** @see validatedAddress */
   address?: string;
 
+  hideXrpBalance = environment.hideXrpBalance;
+
+  tokenIssuer = environment.tokenIssuer;
+
+  tokenSign: string | undefined;
+
+  /** A balance held by the current user. */
+  balances = this.sessionQuery.allBalances;
+
+  paymentOptions?: PaymentOption[];
+
+  checkAmountIsZero?: boolean;
+
   constructor(
     private navCtrl: NavController,
     private modalCtrl: ModalController,
     public sessionService: SessionService,
     public sessionQuery: SessionQuery,
     public notification: SwalHelper,
     public sessionAlgorandService: SessionAlgorandService,
-    public sessionXrplService: SessionXrplService
+    public sessionXrplService: SessionXrplService,
+    public deleteUserSrvice: DeleteUserService,
+    public purePayPageComponent: PurePayPageComponent
```

Comment:
purePayPageComponent not used anywhere so can remove all of it from your delete files

Reply:
Thanks!